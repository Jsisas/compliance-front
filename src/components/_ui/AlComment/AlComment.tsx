import * as React from 'react';
import {Avatar, Comment} from 'antd';

interface AlCommentProps {
	children?: JSX.Element;
}

export function AlComment(props: AlCommentProps): JSX.Element {
	return (
		<Comment
			actions={[<span key="comment-nested-reply-to">Reply to</span>]}
			author={<span>Han Solo</span>}
			avatar={
				<Avatar
					src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
					alt="Han Solo"
				/>
			}
			content={
				<p>
					We supply a series of design principles, practical patterns and high quality design
					resources (Sketch and Axure).
				</p>
			}
		>
			{props.children}
		</Comment>
	);
}
