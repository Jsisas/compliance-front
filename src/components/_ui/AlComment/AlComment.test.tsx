import React from 'react';
import { shallow } from 'enzyme';
import { AlComment } from './AlComment';

describe('<AlComment />', () => {
	it('it renders a comment section', () => {
		const wrapper = shallow(<AlComment />);
		expect(wrapper).toMatchSnapshot();
	});
});
