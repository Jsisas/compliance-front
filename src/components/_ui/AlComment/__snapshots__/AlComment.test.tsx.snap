// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AlComment /> it renders a comment section 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="ant-comment"
      >
        <div
          class="ant-comment-inner"
        >
          <div
            class="ant-comment-avatar"
          >
            <span
              class="ant-avatar ant-avatar-circle ant-avatar-image"
            >
              <img
                alt="Han Solo"
                src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
              />
            </span>
          </div>
          <div
            class="ant-comment-content"
          >
            <div
              class="ant-comment-content-author"
            >
              <span
                class="ant-comment-content-author-name"
              >
                <span>
                  Han Solo
                </span>
              </span>
            </div>
            <div
              class="ant-comment-content-detail"
            >
              <p>
                We supply a series of design principles, practical patterns and high quality design resources (Sketch and Axure).
              </p>
            </div>
            <ul
              class="ant-comment-actions"
            >
              <li>
                <span>
                  Reply to
                </span>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="ant-comment"
    >
      <div
        class="ant-comment-inner"
      >
        <div
          class="ant-comment-avatar"
        >
          <span
            class="ant-avatar ant-avatar-circle ant-avatar-image"
          >
            <img
              alt="Han Solo"
              src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
            />
          </span>
        </div>
        <div
          class="ant-comment-content"
        >
          <div
            class="ant-comment-content-author"
          >
            <span
              class="ant-comment-content-author-name"
            >
              <span>
                Han Solo
              </span>
            </span>
          </div>
          <div
            class="ant-comment-content-detail"
          >
            <p>
              We supply a series of design principles, practical patterns and high quality design resources (Sketch and Axure).
            </p>
          </div>
          <ul
            class="ant-comment-actions"
          >
            <li>
              <span>
                Reply to
              </span>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
